### ISSUE TITLE
RFC: Naming of build-version, deploy-version and deploy-code.

### ISSUE THREAD

**Issue by askreet** [acquia/moonshot#131]
I've been using Moonshot extensively for our project in the last 3 months and I've found that the naming of these commands is overly verbose, and the conflict between the short forms of `deploy-version` and `deploy-code` is annoying.

I'm looking for feedback on renaming, or aliasing, these commands to:
```
build-version -> build
deploy-version -> deploy
deploy-code -> dev-deploy, devdeploy, localdeploy, ???
```

I can't think of a perfect answer for `deploy-code`, but when managing environments across out service delivery pipeline, I definitely wish the first two were as describe above.

**Comment by pdrakeweb** [comment]
I would suggest renaming deploy-code to deploy and leaving deploy-version and build-version alone.  These could have short aliases via map (d, v and b respectively?).  deploy-code could be left as an alias for backward compatibility.

**Comment by askreet** [comment]
@pdrakeweb I don't love the one-letter options, I feel they're really easy to typo. Honestly, as pedantic as it sounds, I'd feel a bit better with two-letter options.

That said, I was trying to make the commands used during the movement of a version through post-development environments shorter, not `deploy-code`.

What about this?
```
build-version -> build
deploy-version -> deploy
deploy-code -> push
```

In all cases, leave the previous full names as aliases, no backwards compatibility is broken.

**Comment by pdrakeweb** [comment]
I like one-letter command aliases but don't feel strongly about them.  95% of the time when I actually *type* a moonshot command it is for development purposes (non-versioned) and so it is most important to me that those be short.

Is there a good reason to change these in Moonshot vs folks simply mapping things how they like in their own CLI tool classes?  Of course, these short command names don't preclude that.

**Comment by askreet** [comment]
@pdrakeweb I think a good reason is simplifying mobility between teams using the same SDLC tooling. It's also a reason I want to revive #89.

**Comment by askreet** [comment]
This was addressed in #139
